### PR TITLE
chore: [#176614756] Use @pagopa packages from npm registry

### PR DESCRIPTION
**Pull request by balanza**
## Short description
Remove the need to use authenticated package registry on Github by switching to public, default Npm registry.
This task is part of a broader activity to move all packaged developed by us to our Npm organization, and to have them scoped under `@pagopa` namespace.

## List of changes proposed in this pull request
- remove `.npmrc` pointing to Github registry
- update `@pagopa/react-native-cie`. Please note that version 1.x has no differences with the previous 0.x but the registry on which is published. 

## How to test
Passing build on both Android and iOS would be enough: code isn't changed, so I assume it either builds or not.
Then, we might want to manually test CIE auth.

### CODE REVIEW

**Comment by Undermaken** [comment]
@balanza Could you update the file [README.md](https://github.com/pagopa/io-app/blob/master/README.md) `Dependencies / module for CIE authentication IO` too?

**Comment by balanza** [comment]
> @balanza Could you update the file [README.md](https://github.com/pagopa/io-app/blob/master/README.md) `Dependencies / module for CIE authentication IO` too?

done

File: README.md [hunk 1]
```diff
@@ -488,7 +488,7 @@ _Note: The sample configuration sets the app to interface with our test environm
 #### Dependencies
 
 **module for CIE authentication**
-IO uses a [react native module](https://github.com/pagopa/io-cie-android-sdk) to allow authentication through CIE (Carta di Indentità Elettronica)
+IO uses a [react native module](https://github.com/pagopa/io-cie-sdk) to allow authentication through CIE (Carta di Indentità Elettronica)
 This package is hosted on [Github Packages](https://github.com/features/packages). In order to install this package you need to be able to access the github registry.
```
Comment (Undermaken) [inline README.md:492]:
```suggestion
```
Could also these sentences be removed?

**Review by Undermaken — CHANGES_REQUESTED** [review]

**Comment by Undermaken** [comment]
@balanza 
maybe should we also update the `packages.json`

from `"@pagopa/react-native-cie": "0.3.3"`
to `"@pagopa/react-native-cie": "1.0.2"`

Schermata 2021-02-01 alle 18 06 42

**Comment by balanza** [comment]
yes of course

**Review by Undermaken — CHANGES_REQUESTED** [review]
@balanza it seems to miss several files when install this package via npm

Schermata 2021-02-01 alle 18

**Comment by balanza** [comment]
Thanks for the screenshot, it's very helpful. I think I fixed it in https://github.com/pagopa/io-app/pull/2754/commits/4b585edd1e7dd5b38570bc656181537dbbf0ca6e, please double check.

**Review by Undermaken — APPROVED** [review]
@balanza Thank you, it works!
Could you double check my last commit https://github.com/pagopa/io-app/pull/2754/commits/42f6a34ab57b5a86288527e6c1a369d642ea3f35 ?

**Pull request merged**